### DISCUSSION TITLE
AreaChartFillByValue but relative to RefernceLine

### DISCUSSION THREAD

**Discussion by ILikeHtmlandCSS** [recharts/recharts#3762] [Q&A] [answered]
Hi,
I recently stumbled about this 
https://recharts.org/en-US/examples/AreaChartFillByValue
in the docs and wonderd if I could achieve the same effect of having everything above 0 being green and everything below it being red but instead of doing it relative to Y 0 to a custom value.

This is my current Graph:
<img width="490" alt="image" src="https://github.com/recharts/recharts/assets/124368951/2d3a5819-1c13-4d8b-bc7a-d6ccfa21b1db">

And this is what I want to achieve:
<img width="553" alt="image" src="https://github.com/recharts/recharts/assets/124368951/aba2913c-0b79-4f14-a337-03f65eb32406">

So as you can see I want to achieve the same effect as in the docs but relative to the my refernce line which just shows the avg of my values.

Thanks for Help!

**Comment by ILikeHtmlandCSS** [comment, marked as answer]
Just got it working using the baseValue prop on the Area element!

`<Area
                                type="monotone"
                                dataKey="time"
                                animationDuration={1000}
                                dot={true}
                                baseValue={avg}
                                fill="url(#colorTime)"
                            />`

Result:
<img width="537" alt="image" src="https://github.com/recharts/recharts/assets/124368951/417f33ec-e49f-45c8-a6a1-2f78d3597e6b">

**Reply by ckifer** [reply]
Sounds good, sorry I couldn't get to reply before you solved it yourself!
